### DISCUSSION TITLE
Floating Start Menu like the Start Menu in windows 11.

### DISCUSSION THREAD

**Discussion by phpjunkie420** [Open-Shell/Open-Shell-Menu#1286] [Ideas]
Here is a screenshot of what I'm talking about.
![Screenshot_20230111_101201](https://user-images.githubusercontent.com/60593808/212001085-bfd8f802-0ab2-453f-ad2f-75325f35d859.png)

This is why I'm suggesting it. I want to make my start menu button bigger and being able to adjust an option to float the start menu would allow me to do that without the button blocking the bottom of the start menu.
![Screenshot_20230111_075822](https://user-images.githubusercontent.com/60593808/212000932-fd46e5fe-4f1a-48c1-8d9e-b118006125fb.png)

**Comment by ge0rdi** [comment]
There is `Horizontal position offset` and `Vertical position offset` in `Menu Look` tab of settings.
You can adjust position of your menu this way.

**Reply by phpjunkie420** [reply]
Thank you
![Screenshot_20230113_040328](https://user-images.githubusercontent.com/60593808/212326706-4988980b-210c-4915-bf8b-d653749594c9.png)
